File: 2011/CCC 2011 S1/S1.py
rows = int(input())

text = ""

for i in range(rows):
    text += input()

t = 0 
s = 0

t += text.count("t") + text.count("T")

s += text.count("s") + text.count("S")

if (t > s):
    print("English")
else:
    print("French")

File: 2011/CCC 2011 S1/S1.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    int n;
    cin >> n;
    cin.ignore(numeric_limits<streamsize>::max(), '\n');

    string text = "";

    for (int i = 0; i < n; i++){
        string temp;
        getline(cin, temp);
        text+=temp;
    }
    
    int counterE, counterF = 0;

    counterE += count(text.begin(), text.end(), 't');
    counterE += count(text.begin(), text.end(), 'T');

    counterF += count(text.begin(), text.end(), 's');
    counterF += count(text.begin(), text.end(), 'S');

    if (counterE > counterF){
        cout << "English";
    } else{
        cout << "French";
    }

    return 0;
}